extensions/MagicChunks.yml

```diff
@@ -2,13 +2,13 @@ Type: Addin
 Name: MagicChunks
 NuGet: MagicChunks
 Assemblies:
-- "/**/MagicChunks.dll"
```

Comment:
This should not be removed as it adds useful information to the API documentation. E.g. [TransformationCollection type](https://cakebuild.net/api/MagicChunks.Cake/MagicChunksAliases/3E7AA7AA) here.

Reply:
AddinDiscoverer was designed to only include one assembly in the YAML.

The particular example you link to is in fact in the assembly the AddinDisco included in the YAML as you can see here:
![image](https://user-images.githubusercontent.com/112710/100801521-14457600-33f6-11eb-842e-ca4d095a7561.png)

Reply:
Correct me if I'm wrong but all the methods decorated with the `[CakeMethodAlias]` attribute are in the one assembly that AddinDisco included in the YAML.

Reply:
Yes, the aliases are all in this assembly, but issue is that they use types in the signature which are in `MagicChunks.dll`. If we don't list it here then these types won't be part of the API documentation (see TransformationCollection type in the link above).

If addin discoverer is not able to handle this, another approach would be to just add a second xml file here which contains MagicChunks.dll just for API documentation.